src/App.tsx

```diff
@@ -1,26 +1,20 @@
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
-import { Loader, TodoFilter, TodoList, TodoModal } from './components';
+import { Todos } from './components/Todos/Todos';
+import { TodoModal } from './components';
+import { useAppSelector } from './hooks/useAppSelector';
 
-export const App = () => (
-  <>
-    <div className="section">
-      <div className="container">
-        <div className="box">
-          <h1 className="title">Todos:</h1>
+export const App = () => {
```

Comment:
The task description specifies that todos should be loaded on page load. Currently, there is no implementation for loading todos when the `App` component mounts. Consider using a hook like `useEffect` to fetch and load todos when the component is first rendered.